cli/popper/commands/cmd_dot.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import click
+import hcl
+import popper.utils as pu
+from ..cli import pass_context
+import types
+
+
+@click.argument(
+    'wfile', required=True)
+@click.command('dot', short_help='Generates a dot file '
+                                 '[Used for graphical representations]')
+@pass_context
+def cli(ctx, wfile):
+    """Creates a dot file"""
+    if not os.path.isfile(wfile):
+        pu.fail("File {} not found.\n".format(wfile))
+        exit(1)
+
+    with open(wfile, 'r') as fp:
+        wf = hcl.load(fp)
```

Comment:
I think we are OK if we print to `stdout` instead to a file. This makes it compatible with Unix conventions, and if users want to store it in a file they can do `popper dot > foo.dot`. Also, printing to `stdout` allows users to pipe and do things like `popper dot | dot -png -o wf.png`

Reply:
Agreed. Will fix it.